code/index.html

```diff
@@ -1,30 +1,141 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Shutter speed</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+<body>
+  <header class="header">
+    <!-- Logo and navbar -->
+    <nav class="nav">
+      <div class="logo-container">
+        <img class="logo" src="./assets/shutter.svg" alt="logo">
+        <p class="logo-name">SHUTTER SPEED</p>
+      </div>
+      <div class="nav-item">
+        <a class="nav-item-links" href="#">Categories</a>
+        <a class="nav-item-links" href="#">Community</a>
+        <a class="nav-item-links" href="#">About</a>
+        <a class="nav-item-signin" href="#">Sign in</a>
       </div>
+    </nav>
+    <!-- Hero section (change images later)-->
+    <picture class="picture">
+      <source srcset="./assets/hero_small.jpg" media="(max-width:767px)">
+      <source srcset="./assets/hero_medium.jpg" media="(min-width:1199px)">
+      <img class="hero-image" src="./assets/hero_original.jpg" alt="Hero image" />
+    </picture>
+    <div class="hero-content">
+      <h1 class="hero-text">Discover the world’s top photographers & creatives</h1>
+      <button class="hero-button">Start exploring</button>
+    </div>
+  </header>
+  <main class="main">
+    <!-- Signup form -->
+    <section class="form-container">
+      <form class="form" action="https://httpbin.org/anything" method="POST">
+        <h2 class="form-heading"> Join Shutter Speed</h2>
+        <p class="form-text">Already have an account? <a href="#">Signin</a></p>
+        <div class="label-container">
+          <label class="label">
+            Name <input class="input-name" type="text" name="form-name" required></input>
+          </label>
+          <label class="label" name="form-username">
+            Username <input class="input-username" type="text" name="form-username" required></input>
+          </label>
+        </div>
+        <label class="label">
+          Email <input class="input" type="email" name="form-email" required></input>
+        </label>
+        <label class="label">
+          Password <input class="input" type="password" name="form-password" placeholder="6+ characters"
```

Comment:
Take a look at the `minlength` attribute [here](https://www.w3schools.com/tags/att_input_minlength.asp).